modules/apps/headless/headless-asset-library/headless-asset-library-test/src/testIntegration/java/com/liferay/headless/asset/library/resource/v1_0/test/AssetLibraryResourceTest.java

```diff
@@ -29,6 +32,15 @@
 @RunWith(Arquillian.class)
 public class AssetLibraryResourceTest extends BaseAssetLibraryResourceTestCase {
 
+	@Before
```

Comment:
Seems to be a workaround to a wrongly generated code.

I've check taht with @AliciaGarciaGarcia and we've seen that it could be fixed changing the ftl in com/liferay/portal/tools/rest/builder/dependencies/base_resource_test_case.ftl:544 adding:

```
<#if stringUtil.equals(javaMethodParameter.parameterName, "keywords")>
							null
```

Reply:
done:

I contacted headless team https://liferay.slack.com/archives/C5E1CRLJY/p1740492617288829 for their input